setup.py

```diff
@@ -92,26 +92,19 @@ def update_version(version):
 
 # Extensions. Add your extension here:
 raw_extensions = [Extension("hyperspy.io_plugins.unbcf_fast",
-                            ['hyperspy/io_plugins/unbcf_fast.pyx']),
+                            [os.path.join('hyperspy', 'io_plugins', 'unbcf_fast.pyx')]),
                   ]
 
 cleanup_list = []
 for leftover in raw_extensions:
     path, ext = os.path.splitext(leftover.sources[0])
     if ext in ('.pyx', '.py'):
-        cleanup_list.append(os.path.join(setup_path, path + '.c*'))
+        cleanup_list.append(''.join([os.path.join(setup_path, path), '.c*']))
```

Comment:
@francisco-dlp Why did you change from `path + 'c*'` to `"".join` in this case?

Reply:
Sorry, I can't remember doing this. The main difference is appending the extension after computing the join path which, in the case of the path to a file, should be the same.